projects/1.20/assets/create-radars/create_radar/lang/zh_cn.json

```
@@ -0,0 +1,43 @@
+{
+  "block.create_radar.auto_pitch_controller": "自动俯仰控制器",
+  "block.create_radar.auto_yaw_controller": "自动偏航控制器",
+  "block.create_radar.monitor": "监视器",
+  "block.create_radar.radar_bearing": "雷达轴承",
+  "block.create_radar.radar_dish_block": "雷达天线",
+  "block.create_radar.radar_plate_block": "雷达板",
+  "block.create_radar.radar_receiver_block": "雷达接收器",
+  "create_radar.display_source.pitch_controller": "自动俯仰控制器",
+  "create_radar.display_source.radar": "雷达",
+  "create_radar.display_source.yaw_controller": "自动偏航控制器",
+  "create_radar.filter.all_entities": "所有实体",
+  "create_radar.filter.mob_bosses_only": "仅BOSS",
+  "create_radar.filter.no_mobs": "非敌对生物",
+  "create_radar.filter.only_mobs": "仅敌对生物",
+  "create_radar.filter.players_only": "仅玩家",
+  "create_radar.filter.projectiles_only": "仅投射物",
+  "create_radar.filter.show": "显示",
+  "create_radar.filter.vs2_only": "仅VS2",
+  "create_radar.ponder.controller_linking.header": "将控制器链接到监视器！",
+  "create_radar.ponder.controller_linking.text_1": "将偏航控制器放置在炮塔底座下方",
+  "create_radar.ponder.controller_linking.text_2": "使用显示链接进行链接",
+  "create_radar.ponder.controller_linking.text_3": "对俯仰控制器重复此操作",
+  "create_radar.ponder.controller_linking.text_4": "在监视器上选择目标，以便炮塔瞄准/射击",
+  "create_radar.ponder.radar_contraption.header": "创建雷达！",
+  "create_radar.ponder.radar_contraption.text_1": "放置雷达轴承",
+  "create_radar.ponder.radar_contraption.text_2": "放置雷达接收器",
+  "create_radar.ponder.radar_contraption.text_3": "添加雷达天线",
+  "create_radar.ponder.radar_contraption.text_4": "雷达天线和雷达板可以混合使用",
+  "create_radar.ponder.radar_contraption.text_5": "额外的天线/板可以增加范围",
+  "create_radar.ponder.radar_contraption.text_6": "为雷达轴承提供应力",
+  "create_radar.ponder.radar_linking.header": "将雷达链接到监视器！",
+  "create_radar.ponder.radar_linking.text_1": "放置监视器",
+  "create_radar.ponder.radar_linking.text_2": "构建雷达装置",
+  "create_radar.ponder.radar_linking.text_3": "使用显示链接进行链接",
+  "create_radar.ponder.radar_linking.text_4": "右键单击显示链接以激活",
+  "create_radar.ponder.tag.radar_components": "雷达",
+  "create_radar.ponder.tag.radar_components.description": "允许创建雷达跟踪系统的组件",
+  "item.create_radar.guided_fuze": "制导引信",
+  "itemGroup.create_radar.radar": "机械动力: 雷达"
```

03.标点符号 错误
```suggestion
  "itemGroup.create_radar.radar": "机械动力：雷达"
```
注意使用中文标点